### DISCUSSION TITLE
Looking for info<title>

### DISCUSSION THREAD

**Discussion by Andy6064** [community/community#42011] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hey all!

A friend suggested this site to find something I'm looking for. I've been trying to find some kind of software containing NLP that would help me find similarities between two or more texts by comprehending the meaning of sentences/paragraphs and matching these with the meaning of sentences/paragraphs in the other text or texts. For example, it would show similarities between two books, article, etc..

Another software I've been looking for is similar to the one explained above but understanding audio, its meaning and finding similarities with a different audio. For example, it would show similarities between two voice messages, podcasts or ted talks.

Any information would be helpful

Thank you in advance

**Comment by getlost01** [comment]
There are several software tools that can help you find similarities between texts or audio files by analyzing their meaning and content. Here are a few options that you may want to consider:

1. **Gavagai Explorer**: This is a tool for analyzing and comparing texts using natural language processing (NLP) technology. It can help you find similarities between texts by extracting key themes and concepts and comparing them to other texts.
2. **IBM Watson**: IBM Watson is a suite of artificial intelligence (AI) tools that includes natural language processing capabilities. It can help you find similarities between texts or audio files by analyzing the meaning and content of the texts or audio and comparing them to other texts or audio. 
3. **Google Cloud Natural Language**: This is a cloud-based natural language processing service that can help you find similarities between texts or audio files by analyzing the meaning and content of the texts or audio, and comparing them to other texts or audio.
4. **Semantria**: This is a tool for analyzing and comparing texts using natural language processing. It can help you find similarities between texts by extracting key themes and concepts and comparing them to other texts.
